tests/conda_env/test_cli.py

```diff
@@ -243,7 +244,7 @@ def test_create_env(self):
         run_conda_command(Commands.CREATE, test_env_name_2)
         self.assertTrue(env_is_created(test_env_name_2))
 
-    def test_export(self):
+    def test_export_1(self):
```

Comment:
What does the _1 denote here?

Reply:
To distinguish from https://github.com/conda/conda/pull/6221/files#diff-771b53ed2768cb3fe7eb0bc279a57c34R297

That is, with

    py.test tests/conda_env/test_cli.py -k test_export

there was no way to specify only the one test before.

Reply:
Did that make any sense at all? 🙃

Reply:
Now, with 

    py.test tests/conda_env/test_cli.py -k test_export

you get the two tests, but if you want to target that single test, 

        py.test tests/conda_env/test_cli.py -k test_export_1

So it's just about name uniqueness.

Reply:
Ah, okay.

Reply:
Well, I wouldn't be too pedantic and ask it to be renamed to `test_env_export` or something.

Reply:
There ya go :)